backend/courses/models.py

```diff
@@ -433,6 +433,145 @@ def __str__(self):
         return f"Topic {self.id} ({self.most_recent.full_code} most recently)"
 
 
+class Attribute(models.Model):
+    """
+    A post-NGSS registration attribute, which is used to
+    mark courses which students in a program/major should take.
+    e.g. WUOM for the "Wharton OIDD Operation" track
+
+    Note that Attributes (like Restrictions) do not have an associated
+    semester.
+    """
+
+    code = models.CharField(
+        max_length=10,
+        unique=True,
+        db_index=True,
+        help_text=dedent(
+            """
+        A registration attribute code, for instance 'WUOM' for Wharton OIDD Operations track.
+        """
+        ),
+    )
+
+    description = models.TextField(
+        help_text=dedent(
+            """
+        The registration attribute description, e.g. 'Wharton OIDD Operation'
+        for the WUOM attribute.
+        """
+        )
+    )
+
+    SCHOOL_CHOICES = (
+        ("SAS", "School of Arts and Sciences"),
+        ("LPS", "College of Liberal and Professional Studies"),
+        ("SEAS", "Engineering"),
+        ("DSGN", "Design"),
+        ("GSE", "Graduate School of Education"),
+        ("LAW", "Law School"),
+        ("MED", "School of Medicine"),
+        ("MODE", "Grade Mode"),
+        ("VET", "School of Veterinary Medicine"),
+        ("NUR", "Nursing"),
+        ("WH", "Wharton"),
+        ("OTHER", "Other"),
```

Comment:
Sorry just noticed this -- does this actually represent "none of the above", or is it more like "unknown"? From `identify_school` it seems like the latter? In that case the name "OTHER" might be misleading -- we might want to make this field nullable and let null represent the case where we don't know what the school is (or the attribute isn't associated with a school).